actionpack/lib/action_dispatch/routing.rb

```diff
@@ -120,9 +120,9 @@ module ActionDispatch
   #
   #   # In config/routes.rb
   #   controller :blog do
-  #     get 'blog/show',    to: :list
-  #     get 'blog/delete',  to: :delete
-  #     get 'blog/edit',    to: :edit
+  #     get 'blog/show',    to: 'blog#list'
+  #     get 'blog/delete',  to: 'blog#delete'
+  #     get 'blog/edit',    to: 'blog#edit'
```

Comment:
When using the `controller` method, there is no need to use `to:` with the controller and action, as the controller is already defined.
I think we should revert this to the old hash notation and document this notation is possible as well:

```suggestion
  #     get 'blog/show' => :list
  #     get 'blog/delete' => :delete
  #     get 'blog/edit' => :edit
```

Reply:
Agree. Can you make the change please?

Reply:
Done. Verified:

```ruby
routes.rb:

   controller :blog do
      get 'blog/show'    => :list
      get 'blog/delete'  => :delete
      get 'blog/edit'    => :edit
    end
		
 ➜ /workspaces/rails-docs (main) $ ruby -v
ruby 3.3.0 (2023-12-25 revision 5124f9ac75) [x86_64-linux]
 ➜ /workspaces/rails-docs (main) $ rails -v
Rails 7.1.2
 ➜ /workspaces/rails-docs (main) $ rails routes | grep blog
                               blog_show GET  /blog/show(.:format)      blog#list
                             blog_delete GET  /blog/delete(.:format)    blog#delete
                               blog_edit GET  /blog/edit(.:format) 			blog#edit
```

Reply:
This:

```ruby
   controller :blog do
      get 'blog/show',    to: :list
    end
```

will not work (current docs). The error:

```
rails routes | grep blog
/usr/local/rvm/gems/ruby-3.3.0@dox/gems/actionpack-7.1.2/lib/action_dispatch/routing/mapper.rb:312:in `split_to': undefined method `include?' for an instance of Symbol (NoMethodError)

            if to&.include?("#")
                 ^^^^^^^^^^
```